### PR TITLE
fix - Fixes endpoints UI on save and on query run bugs

### PR DESCRIPTION
**Pull request by filipecabaco**
* On save, the lack of a default language on Query creation made a changeset fail
* On query test, the input would be deleted since the value was not being set and assigns were being removed

> Note: Endpoints are behaving weirdly due to other reasons in my dev environment so take that into account in the gifs below 😅 

## Before
before
## After
after

### CODE REVIEW

File: lib/logflare_web/live/endpoints/endpoints_live.ex [hunk 1]
```diff
@@ -190,24 +178,18 @@ defmodule LogflareWeb.EndpointsLive do
...
-    {:noreply,
-     socket
-     |> put_flash(:info, "Successfully applied for the Endpoints beta. We'll be in touch!")}
+    message = "Successfully applied for the Endpoints beta. We'll be in touch!"
+    {:noreply, put_flash(socket, :info, message)}
   end
 
-  defp update_params_form(socket, parameters, query_string) do
+  defp assign_updated_params_form(socket, parameters, query_string) do
+    params = for(k <- parameters, do: {k, nil}, into: %{})
```
Comment (Ziinc) [inline lib/logflare_web/live/endpoints/endpoints_live.ex:186]: this would still clear the form values, i think.

**Review by Ziinc — COMMENTED** [review]
could we have a screencap of the fix if it can't be tested easily

**Comment by filipecabaco** [comment]
Posted a GIF with the before and after

File: lib/logflare_web/live/endpoints/endpoints_live.ex (hunk 1 above)
Comment (filipecabaco) [reply lib/logflare_web/live/endpoints/endpoints_live.ex:186]: No, all good 👍 works as expected gifs in the PR

**Review by Ziinc — APPROVED** [review]
Awesome thanks @filipecabaco 👍😁

**Pull request merged**